### DISCUSSION TITLE
👋 Welcome to Batch#6 - Introductions!

### DISCUSSION THREAD

**Discussion by edakturk14** [BuidlGuidl/batch6.buidlguidl.com#1] [Announcements]
Hey everyone!

It's Eda here, I've been part of the BuidlGuidl for almost 1.5 years! I'm a developer and I spend a lot of time [writing](https://mirror.xyz/edatweets.eth) about different topics and concepts in crypto. I enjoy working on developer education and creating content/documentation.

Excited to have you onboard for our 6th Batch program! If you have any questions, feel free to ask anytime!

**Comment by carletex** [comment]
Hey all!! 

My name is Carlos, I've been part of the BuidlGuidl since 2021. I've been a software developer for more than 10 years and went _onchain_ three years ago.

During this time I've built things like the [SpeedRunEthereum](https://speedrunethereum.com/) platform, the [BuidlGuidl App](https://app.buidlguidl.com/), [ABI Ninja](https://abi.ninja/) & [Scaffold-ETH 2](https://github.com/scaffold-eth/scaffold-eth-2). Inside the BuidlGuild, I also run the [Sand Garden](https://sandgarden.buidlguidl.com/).

I love learning and building things 👾

Alongside other BG members, I'll be helping you all to go through this batch. Hope you all have fun!

**Comment by jriyyya** [comment]
Hi Everyone, I am Riya Jain, Currently in my final year, doing Btech in Information technology.

I am a full stack developer but I contribute significantly to the Frontend, I have participated and won a few of the Hackathons, organised by Chainlink, Avalanche, HomeDao, TBD etc.

I am also an active open source contributor, I successfully completed my Google summer of code (GSoC'23) project under the Sugar Labs organisation and currently on my project of GSoC'24 under the organisation Zendalona.

I really like the web3 community and I am really excited about this! Happy to connect with you all.

LinkedIn: https://linkedin.com/in/jriyyya

**Reply by carletex** [reply]
Welcome Riya! 

Wow, You are very active! Happy to have you here, hope you stick around and get to know the BG a bit more.

**Comment by marsian83** [comment]
Hi Guys, I am Spandan, I also go by marsian, I am currently in my final year, pursuing a Bachelors in Computer Science Engineering.

I have been into development for about a decade now, I got started developing games just for fun 😁, I have been exploring the  web3 space for over 2 years now and I'm an avid learner.

I have participated and won various web3 hackathons, organised by EthGlobal, Chainlink, Avalanche, TBD, Homedao, etc.
@jriyyya and I mostly work together, we participate in hacakathons as part of the same team which is why our participations/achievemnts are so similar. 

I am also active in the open source community, I was selected and have completed my Linux Mentorship program, LFX'23 under the organisation Cloud native computing foundation - Vitess. I have also been selected for Google Summer of code (GSoC'24).

You can find me on LinkedIn: https://linkedin.com/marsian83

**Reply by carletex** [reply]
Hey Marsian!! Y'all are killing it with all the hackathon participation. Happy to have you here, hope you learn a lot from the BG!

**Comment by kocaemre** [comment]
Hello everyone, my name is Emre. I am a second-year computer engineering student.

I am improving myself in the fields of cybersecurity and blockchain. I am the admin of the "Airdrop Academy" Telegram group.

I learned about the Buidl Guidl community through a university event. Also, speedrunethereum has improved my coding skills.

I am looking forward to creating wonderful projects.

Linkedin : https://www.linkedin.com/in/emre-koca-333283253/

**Reply by carletex** [reply]
Selam Emre! I also like the cybersecurity field 👾

Glad you liked SpeedRunEthereum. Hope you can get some learning from the Batch too ;)

**Comment by wildanvin** [comment]
Hello guys!

My name is Wilman, I'm from Ecuador. My background is in electrical engineering but since 2021 I have been learning everything I can about blockchain and Ethereum. 

I have built some personal projects, participated in some hackathons, won some prizes and here I am ready to dev!

One of the things that I love about the space is that you don't need permission to build or do what you want, you are only a few clicks away of interacting with a protocol or code something cool. 

Really happy to be here and collaborate with you all!

This is my LinkedIn: https://www.linkedin.com/in/wilman-vinueza/
but i respond faster on twitter 😅: https://x.com/wildanvin

**Reply by carletex** [reply]
Bienvenido Wilman!!

Happy to have you here, hope we all learn together in this Batch.

> One of the things that I love about the space is that you don't need permission to build or do what you want, you are only a few clicks away of interacting with a protocol or code something cool. 

Same! No need to validate your email, ID, etc.

**Comment by JeFlame** [comment]
Hi everyone!! 👋👋

My name is Hoang but you can call me J. I am a backend developer, and I began my journey in blockchain technology in March 2024.

I love spending time learning and building web3 projects.

I am really happy to be here, and I hope that we will have a memorable time together!

My LinkedIn: [https://www.linkedin.com/in/hoang-vu-847288202](https://www.linkedin.com/in/hoang-vu-847288202)

**Reply by carletex** [reply]
What's up J!

Happy to have you here! I'm sure you will learn some frontend skills here too haha. We always say that you spend 30 min building a smart contract and then 4 hours trying to center a div :P

**Reply by JeFlame** [reply]
LOL! I wish I knew the trick to deal with that 'centering the div' too 😂 😂

**Comment by Okhayeeli** [comment]
Hi, My name is Elizabeth khaye, I am graduate of Civil Engineering who transitioned into tech in 2022 after my graduation. I started with learning about AWS and python. I interned at a company where I worked with vanilla javascript, php and MySql. I started my  web3 journey in feb, learning from cyfrin updraft.  I am so grateful to be here, I am pumped to learn a lot and become better. thank you!!!!

**Reply by carletex** [reply]
Welcome Elizabeth!

Love the transition! PHP & Mysql have a special place in my heart (it's where I started!). Cyfrin & Patrick are great resources <3

Hope to learn a ton from the batch!

**Comment by MayowaObisesan** [comment]
Hi everyone 👋 

My name is Mayowa. You can call me Blessed. I am excited to be here. I've been a software developer for 3 years and went onchain in August 2023.
During my time in the web3 space, I have built for different Hackathons. Including Base, Near, Cartesi, Lightlink, including the just concluded Scroll Vortex01 Hackathon. I have won 5 Hackathons from December 2023 till May 2024.

I love to build and I am excited to have begin building on buidlguidl.

I learnt about buidl guidl from a friend and I'm happy to be a part of the community now.

Thank you all for having me. 🙂

You can reach me on:
Twitter: [https://x.com/blessed_mayowa](https://x.com/blessed_mayowa)
Linkedin: [https://linkedin.com/in/mayowa-obisesan](https://linkedin.com/in/mayowa-obisesan)
Telegram: [@BlessedMayowa](https://t.me/BlessedMayowa)
Discord: [@Blessed07]()

**Reply by carletex** [reply]
Welcome Mayowa!! Great hackathon record!! Excited to have you onboard ;)

**Reply by MayowaObisesan** [reply]
Thanks @carletex . Happy to be here.

**Comment by hdevelopergit** [comment]
Hi All! 👋
My name is Hugo. I'm from Buenos Aires - Argentina. Currently living in Barcelona. I'm a system engineer and work as a developer since 20 years ago. I'm specialized in the SAP sector and lastly expierencing with web development (always in SAP, specifically javascript).
So, I don't have experience as a regular web developer 😁 Sorry about that and please be patience with me 😊
I'm interested in blockchain since a while and find this world facinating. I expect be helpful in this batch and learn a lot!

Thanks for accepting me in this journey 😊

**Reply by carletex** [reply]
Bienvenido Hugo! Barcelona is great :P

Happy to have you here! You'll adapt pretty well with all the experience you have haha.

**Reply by hdevelopergit** [reply]
Nice to meet you @carletex . Thanks! :-)
